[BUG] Suiri-13 Fix bug

Fix bug on new user '/start' command

`Caused by: org.postgresql.util.PSQLException: ERROR: null value in column "test_type" of relation "user_settings" violates not-null constraint
suiribot_container |   Detail: Failing row contains (1, f, ru, null, null).
suiribot_container |    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.6.jar:42.3.6]
suiribot_container |    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2365) ~[postgresql-42.3.6.jar:42.3.6]
suiribot_container |    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:355) ~[postgresql-42.3.6.jar:42.3.6]
suiribot_container |    at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490) ~[postgresql-42.3.6.jar:42.3.6]
suiribot_container |    at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408) ~[postgresql-42.3.6.jar:42.3.6]
suiribot_container |    at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:167) ~[postgresql-42.3.6.jar:42.3.6]
suiribot_container |    at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:135) ~[postgresql-42.3.6.jar:42.3.6]
suiribot_container |    at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61) ~[HikariCP-4.0.3.jar:na]
suiribot_container |    at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java) ~[HikariCP-4.0.3.jar:na]
suiribot_container |    at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:197) ~[hibernate-core-5.6.10.Final.jar:5.6.10.Final]
suiribot_container |    ... 129 common frames omitted
suiribot_container |` 
## Acceptance Criteria
Bot must working properly on '/start' command

--- a/RELEASE_NOTES.MD
+++ b/RELEASE_NOTES.MD
@@ -1,6 +1,9 @@
 ## RELEASE NOTES
 
-## 1.7.0-SNAPSHOT
+## 1.8.1-SNAPSHOT
+* Fixed the bug with '/start' command 
+
+## 1.8.0-SNAPSHOT
 * Added gif support to bot
 
 ## 1.7.0-SNAPSHOT

--- a/pom.xml
+++ b/pom.xml
@@ -10,7 +10,7 @@
     </parent>
     <groupId>com.odeyalo.bot</groupId>
     <artifactId>suiri</artifactId>
-    <version>1.8.0-SNAPSHOT</version>
+    <version>1.8.1-SNAPSHOT</version>
     <name>Suiri</name>
     <description>Suiri is a powerful telegram bot to learn new words</description>
     <properties>

--- a/src/main/java/com/odeyalo/bot/suiri/service/command/StartCommandExecutor.java
+++ b/src/main/java/com/odeyalo/bot/suiri/service/command/StartCommandExecutor.java
@@ -58,6 +58,7 @@ public class StartCommandExecutor implements CommandExecutor {
                 .user(user)
                 .enableNotification(false)
                 .language(userLanguage)
+                .testType(UserSettings.PreferredKnowledgeTestType.UPDATABLE_MESSAGE)
                 .build();
         user.setUserSettings(userSettings);
         user.setUserDictionary(dictionary);

